pkg/xds/server/components.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"time"
 
+	"github.com/Kong/kuma/pkg/core/faultinjections"
```

Comment:
I think this should be in a group with other kuma imports. This goimport linter is not ideal unfortunately :(